src/pages/Detail/SectionLeft/Carousel/Carousel.jsx

```diff
@@ -1,24 +1,157 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
-import sample from '../../../../images/sample.png';
 
 const Carousel = () => {
+  const [buttonId, setButtontId] = useState('r1');
+  const moveRight = () => {
+    const changedNum = Number(buttonId.slice(1));
+    setButtontId(`r${changedNum + 1}`);
+    changedNum === RADIOS.length && setButtontId(`r1`);
```

Comment:
- 경우에 따라서 setButtonId 가 2번 호출됩니다. 로직을 통해 그러지 않도록 해줄 수 있을 것 같습니다.
```suggestion
    const changedNum = Number(buttonId.slice(1));
    const newButtonId = changeNum === RADIOS.length ? 'r1' : `r${changedNum + 1}`;
    setButtontId(newButtonId);
```

Reply:
다시 보니 정말 불필요하게 2번 호출될 경우가 있네요! 리뷰 반영했습니다!